optimum/gptq/quantizer.py

```diff
@@ -69,8 +75,8 @@ def __init__(
         module_name_preceding_first_block: Optional[List[str]] = None,
         batch_size: int = 1,
         pad_token_id: Optional[int] = None,
-        disable_exllama: Optional[bool] = None,
-        disable_exllamav2: bool = False,
+        disable_exllama: bool = False,
+        exllama_config: Dict[str, Any] = None,
```

Comment:
It is unclear in the docstring which of the two take priority

Reply:
exllama_config is just the config that is used when `disable_exllama = False`. I will add more info in the docstring.